### ISSUE TITLE
[Adaptive Cards - Elements Properties] - Heading is not defined for the 'Text Block, Layout and Styles' under elements properties pane.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9038]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on the Edge Dev browser.
2. Press tab key and navigate to the Designer tab item and press enter key to select it.
3. Press tab key navigate to the editor section and select any field.
4. Press tab key and navigate to the 'Element Property' button and press enter key to activate it.
5. Navigate the pane using 'H' key and observe the issue.

**Actual Result:**
Under the 'Elements Properties' pane, no heading is defined for the 'Text Block, Layout and styles'. If a screen reader user navigates using h key in heading mode. then it is not announcing the any information about the heading.

**Expected Result:**
'Text Block, Layout and styles' should be defined as heading so that when the screen reader user navigate using h key then it should navigate on each label and announced the heading information.

**Note:**
While in developer mode, Publish Adaptive Card Schema is also not defined as heading.

**User Experience:**
Screen reader users often rely on heading shortcuts to efficiently navigate a page. Without proper headings, they may struggle to quickly locate and access specific sections of content, making their experience less efficient.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Heading is not defined for the 'Text Block, Layout and Styles' under elements properties pane](https://github.com/user-attachments/assets/b9161c42-59a4-415b-a15e-cbe27e793984)

![Heading is not defined for the 'Text Block, Layout and Styles' under elements properties pane using tool](https://github.com/user-attachments/assets/832327a3-e24e-4e66-bb21-a917c54cffe2)

https://github.com/user-attachments/assets/b2a14cfd-f345-4aa8-b957-919cf2ca0ba2

### Card JSON

```JSON
NA
```
